copilot/README.md

```diff
@@ -1,8 +1,12 @@
-# VSCode Copilot
+# Copilot Interaction in VSCode
```

Comment:
Why the title change?

Reply:
I can revert if you want. Just seemed like it was a bit clearer to indicate what the folder is all about. i.e. our gpt folder is titled `GPT and LLM Interaction` so seems like it matched the stye